server/src/main/java/io/druid/segment/loading/SegmentLoaderLocalCacheManager.java

```diff
@@ -270,4 +264,19 @@ public void cleanupCacheFiles(File baseFile, File cacheFile) throws IOException
       }
     }
   }
+
+  public Set<StorageLocation> getSortedSet(List<StorageLocation> locs)
```

Comment:
But this should be a set, right ? Even though `StorageLocation` does not override `equals` and `hashcode` atleast same object instance cannot be added twice as it is a set.